### PR TITLE
tidb: add some simple select benchmark.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by shenli** [comment]
Should we create a new package or a new repo for bench?

**Comment by coocood** [comment]
It's more convenient to include benchmark n this repo.
And there would be a lot of package specific benchmarks.

**Comment by shenli** [comment]
LGTM

File: bench_test.go [hunk 1]
```diff
@@ -1,30 +1,172 @@
...
+			b.Fatal(err)
+		}
+		readResult(rs[0], 1)
+	}
+}
+
+func BenchmarkIntegerIndexScan(b *testing.B) {
+	b.StopTimer()
+	se := prepareBenchSession()
+	prepareBenchData(se, "int", "%v", 100)
```
Comment (qiuyesuifeng) [inline bench_test.go]: Using smallCount?

File: bench_test.go [hunk 2]
```diff
@@ -1,30 +1,172 @@
...
+	b.StopTimer()
+	se := prepareBenchSession()
+	prepareBenchData(se, "decimal(32,6)", "%v.1234", smallCount)
+	b.StartTimer()
+	for i := 0; i < b.N; i++ {
+		rs, err := se.Execute("select * from t where col >= 0")
+		if err != nil {
+			b.Fatal(err)
+		}
+		readResult(rs[0], 100)
```
Comment (qiuyesuifeng) [inline bench_test.go]: Using smallCount

**Comment by zimulala** [comment]
LGTM

**Pull request merged**